ui/src/App.tsx

```diff
@@ -206,8 +211,10 @@ function App() {
           </p>
         </Container>
       )}
-      {activityMain}
-    </>
+      <Routes >
```

Comment:
nit: the extra space before the `>` made me do a double-take (my brain parsed it as a greater-than operator rather than the end of the HTML tag)

Reply:
Fixed